db/seeds.rb

```diff
@@ -18,8 +18,94 @@
   timezone: "Central Time (US & Canada)"
 )
 
-business = Business.create!(
+business_happy_seedlings = Business.create!(
   name: "Happy Seedlings Childcare",
   category: "licensed_center_single",
   user: user
-)
\ No newline at end of file
+)
+
+site_happy_seeds_little_oaks = Site.create!(
```

Comment:
Check the seeds I've got in the #175 PR; I've shifted from `create!` and have a few sites seeded.

Reply:
Will do.

Reply:
Just curious:  why not the `!` version of `first_or_create`?  Why not use `first_or_create!` so you explicitly get an error if the create fails?

Reply:
Mostly because seeding is a non-typical action, but that's a valid point, do you mind tweaking them across the whole seed file?